$OpenBSD: patch-configure,v 1.7 2010/09/30 07:26:05 ajacoutot Exp $

XXX make it compile with gtk+2 2.22 (*_DISABLE_DEPRECATED)
kz-tabtree.c:88: error: expected declaration specifiers or '...' before 'GtkNotebookPage'
kz-tabtree.c:699: error: expected declaration specifiers or '...' before 'GtkNotebookPage'

XXX we know our xl has webprogresslistener2 but nspr headers not in -I during configure

--- configure.orig	Tue Sep 29 05:44:00 2009
+++ configure	Wed Sep 29 20:09:55 2010
@@ -16296,13 +16296,13 @@ fi
 
   if test x"$enable_gtkmozembed" = "xyes"; then
     if test x"$GECKO" = "xlibxul"; then
-      GECKO_MODULE=$GECKO-unstable
+      GECKO_MODULE=$GECKO
     else
       GECKO_MODULE=$GECKO-xpcom
     fi
   else
     if test x"$GECKO" = "xlibxul"; then
-      GECKO_MODULE=$GECKO-embedding-unstable
+      GECKO_MODULE=$GECKO-embedding
     else
       GECKO_MODULE=$GECKO-gtkmozembed
     fi
@@ -16669,14 +16669,9 @@ nsIWebProgressListener2 *listener;
   return 0;
 }
 _ACEOF
-if ac_fn_cxx_try_compile "$LINENO"; then :
 
 $as_echo "#define HAVE_NSIWEBPROGRESSLISTENER2_ONREFRESHATTEMPTED 1" >>confdefs.h
 
-         result=yes
-else
-  result=no
-fi
 rm -f core conftest.err conftest.$ac_objext conftest.$ac_ext
 
 { $as_echo "$as_me:${as_lineno-$LINENO}: result: $result" >&5
@@ -20796,77 +20791,6 @@ fi
 
 anthy_exists=no
 
-pkg_failed=no
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for ANTHY" >&5
-$as_echo_n "checking for ANTHY... " >&6; }
-
-if test -n "$PKG_CONFIG"; then
-    if test -n "$ANTHY_CFLAGS"; then
-        pkg_cv_ANTHY_CFLAGS="$ANTHY_CFLAGS"
-    else
-        if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"anthy\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "anthy") 2>&5
-  ac_status=$?
-  $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
-  test $ac_status = 0; }; then
-  pkg_cv_ANTHY_CFLAGS=`$PKG_CONFIG --cflags "anthy" 2>/dev/null`
-else
-  pkg_failed=yes
-fi
-    fi
-else
-	pkg_failed=untried
-fi
-if test -n "$PKG_CONFIG"; then
-    if test -n "$ANTHY_LIBS"; then
-        pkg_cv_ANTHY_LIBS="$ANTHY_LIBS"
-    else
-        if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"anthy\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "anthy") 2>&5
-  ac_status=$?
-  $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
-  test $ac_status = 0; }; then
-  pkg_cv_ANTHY_LIBS=`$PKG_CONFIG --libs "anthy" 2>/dev/null`
-else
-  pkg_failed=yes
-fi
-    fi
-else
-	pkg_failed=untried
-fi
-
-
-
-if test $pkg_failed = yes; then
-
-if $PKG_CONFIG --atleast-pkgconfig-version 0.20; then
-        _pkg_short_errors_supported=yes
-else
-        _pkg_short_errors_supported=no
-fi
-        if test $_pkg_short_errors_supported = yes; then
-	        ANTHY_PKG_ERRORS=`$PKG_CONFIG --short-errors --errors-to-stdout --print-errors "anthy"`
-        else
-	        ANTHY_PKG_ERRORS=`$PKG_CONFIG --errors-to-stdout --print-errors "anthy"`
-        fi
-	# Put the nasty error message in config.log where it belongs
-	echo "$ANTHY_PKG_ERRORS" >&5
-
-	{ $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
-$as_echo "no" >&6; }
-                anthy_exists=no
-elif test $pkg_failed = untried; then
-	anthy_exists=no
-else
-	ANTHY_CFLAGS=$pkg_cv_ANTHY_CFLAGS
-	ANTHY_LIBS=$pkg_cv_ANTHY_LIBS
-        { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
-$as_echo "yes" >&6; }
-	anthy_exists=yes
-fi
-
 # Extract the first word of "mecab-config", so it can be a program name with args.
 set dummy mecab-config; ac_word=$2
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for $ac_word" >&5
@@ -21422,7 +21346,7 @@ fi
 GENHTML_OPTIONS="--title 'Kazehakase Code Coverage'"
 
 KAZEHAKASE_CFLAGS="$GTK_CFLAGS $LIBGNUTLS_CFLAGS $GIO_CFLAGS $DBUS_GLIB_CFLAGS"
-KAZEHAKASE_CFLAGS="$KAZEHAKASE_CFLAGS -DGDK_DISABLE_DEPRECATED -DGTK_DISABLE_DEPRECATED"
+KAZEHAKASE_CFLAGS="$KAZEHAKASE_CFLAGS -DGDK_DISABLE_DEPRECATED"
 KAZEHAKASE_CFLAGS="$KAZEHAKASE_CFLAGS -DG_DISABLE_DEPRECATED"
 KAZEHAKASE_CFLAGS="$KAZEHAKASE_CFLAGS -DSYSDATADIR=\\\"\"\$(datadir)\"\\\""
 KAZEHAKASE_CFLAGS="$KAZEHAKASE_CFLAGS -DLOCALEDIR=\\\"\"\$(localedir)\"\\\""
